app/controllers/concerns/offender_sar_cases_params.rb

```diff
@@ -26,11 +29,14 @@ def create_offender_sar_params
       :third_party_name,
       :third_party_relationship,
       :third_party_company_name,
+      :date_responded_dd, :date_responded_mm, :date_responded_yyyy,
+      :date_of_birth_dd, :date_of_birth_mm, :date_of_birth_yyyy,
```

Comment:
Were these deleted and re-added? I thought it these fields were already in the app?

Reply:
I think I deleted them by accident and re-added